src/utils.ts

```
@@ -213,3 +213,13 @@ export function convertToTelegramButtons(buttons?: Button[] | null): InlineKeybo
   });
 }
+
+/**
+ * Clean text by removing all NULL (\u0000) characters
+ * @param {string | undefined | null} text - The text to clean
+ * @returns {string} The cleaned text
+ */
+export function cleanText(text: string | undefined | null): string {
+  if (!text) return '';
+  return text.replace(/\u0000/g, '');
+}
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)